src/context/data-context.tsx

```diff
@@ -0,0 +1,20 @@
+import { createContext, ReactNode, useContext } from "react";
+import { darkTheme, lightTheme } from "../themes/themes";
+
+interface DataContextInterface {
+  theme: any;
+}
+
+const DataContext = createContext<DataContextInterface>({ theme: lightTheme });
+
+const DataProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
UI looks amazing, Clean code
Only suggestion : change name to ThemeProvideer instead of data Provider